pshtt/pshtt.py

```diff
@@ -748,9 +742,21 @@ def https_check(endpoint):
                 if(PT_INT_CA_FILE is not None):
                     try:
                         cert_plugin_result = None
-                        command = sslyze.plugins.certificate_info_plugin.CertificateInfoScanCommand(ca_file=PT_INT_CA_FILE)
-                        cert_plugin_result = scanner.run_scan_command(server_info, command)
-                        if(cert_plugin_result.verified_certificate_chain is not None):
+                        scanner = Scanner()
+                        command = ScanCommand.CERTIFICATE_INFO
+                        command_extra_args = {
+                            ScanCommand.CERTIFICATE_INFO: CertificateInfoExtraArguments(custom_ca_file=Path(PT_INT_CA_FILE))
+                        }
+                        scan_request = ServerScanRequest(
+                            server_info=server_info,
+                            scan_commands_extra_arguments=command_extra_args,
+                            scan_commands=[command]
+                        )
+                        scanner.queue_scan(scan_request)
+                        # Retrieve results from generator object
```

Comment:
Same rationale as the earlier feedback.

```suggestion
                        # Consume the generator object and retrieve the first result
```